sdk/src/main/java/com/kin/ecosystem/marketplace/presenter/MarketplacePresenter.kt

```diff
@@ -47,6 +48,7 @@ class MarketplacePresenter(private val offerRepository: OfferDataSource,
     private var currentBalance: Balance = blockchainSource.balance
     private var publicAddress: String = blockchainSource.publicAddress
 
+    private var isPageOfferEventSent = AtomicBoolean(false)
```

Comment:
why need atomic here?

Reply:
it can be called from `getOffers()` which is async call, and also from `getCachedOffers()`